helm/templates/secrets.yaml

```diff
@@ -10,16 +9,18 @@ metadata:
     heritage: "{{ .Release.Service }}"
 type: Opaque
 data:
-  {{- if .Values.mongodbRootPassword }}
-  mongodb-root-password:  {{ .Values.mongodbRootPassword | b64enc | quote }}
-  {{- else }}
-  mongodb-root-password: {{ randAlphaNum 10 | b64enc | quote }}
```

Comment:
Removed random password generation since it will be overwritten when any value in secret changes. Using `required` helm function to force user to set this.